worldedit-core/src/main/java/com/fastasyncworldedit/core/math/BlockVectorSet.java

```diff
@@ -27,6 +29,26 @@ public class BlockVectorSet extends AbstractCollection<BlockVector3> implements
 
     private final Long2ObjectLinkedOpenHashMap<LocalBlockVectorSet> localSets = new Long2ObjectLinkedOpenHashMap<>(4);
 
+    public BlockVectorSet() {
+    }
+
+    public BlockVectorSet(LocalBlockVectorSet localSet) {
+        if (localSet != null && localSet.isInitialised()) {
+            int offsetX = localSet.offsetX();
+            int offsetY = localSet.offsetY();
+            int offsetZ = localSet.offsetZ();
+            if ((offsetX & 2048) != 0 && (offsetY & 512) != 0 && (offsetZ & 2048) != 0) { // Can plug and play
```

Comment:
I think I'm looking at this the third time and still questioning my sanity. But I assume this is supposed to check whether the localSet is already (2048, 512, 2048)-aligned? In that case, it would need to be `& 2047 == 0` etc. Otherwise I need an explanation because I can't wrap my head around it.

Reply:
No you're right, I wasn't really thinking when I wrote this. I had meant to come back to look again but forgot and just committed